README.md

```diff
@@ -17,10 +17,9 @@ to be run in: e.g develop, test, staging, and production.  Each service contains
 under the environment key public values for environment variables used as params
 for the API configuration.
 
-Private values used in each environment are stored in Lastpass and are never
-placed in docker-compose.yml and never committed to the repo.  At the root level
-of the repo locally create a file called .env and save the Lastpass contents for
-"bi-api secrets" in this file.
+Private values used in each environment are never placed in docker-compose.yml 
+and never committed to the repo. Variables are passed into the docker-compose.yml
+from the .env file. The .env file should be modified to contain the secrets.
```

Comment:
One thing I found with the different env files is that it has to specified for each service, and can't be a one line change to the docker-compose file for all the services. You also can't specify the new env file in the command line. I wish docker-compose made it easier to do this!

Reply:
Nevermind, Tim sent me the command he had and it worked great! I was using the wrong syntax. The instructions for this will be added to the PR for ONT-68.